tests/test_skip_behavior.py

```
@@ -0,0 +1,54 @@
+import subprocess
+import sys
+
+print("=" * 70)
+print("Testing skip behavior in parallel mode")
+print("=" * 70)
+
+# Test 1: Run with -n0 (should run)
+print("\n1. Running with -n0 (serial mode - should RUN)...")
+result = subprocess.run(
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)